### PR TITLE
AU profiles update

### PR DESCRIPTION
**Pull request by Andrej730**
Updated AU profiles data based on more correct source.
I've also parsed the other data for these profiles too and will be able to update them in the future like I've added inner and outer fillet radius for RHS and SHS after cf748acf77ea969f612bcd0699ee84fde2c5f08a

I've also separated bluescope i-beams to welded and universal becauses welded ones don't have fillet radius.

Sources:
- https://www.steel.org.au/resources/elibrary/resources/design-capacity-tables-for-structural-steel-hollow/ (for hollow profiles)
- One Steel Manufacturing Hot Rolled and Structured Steel Products 7th Ed (pages 12-28).

### CODE REVIEW

**Comment by berndhahnebach** [comment]
Have you seen https://forum.freecadweb.org/viewtopic.php?f=23&t=72744&p=639186#p639132 Do you take care of this? Would you just add this to the PR ?

**Comment by berndhahnebach** [comment]
Have you seen https://github.com/boltsparts/boltsparts/pull/4 Do you take care of this? Would you just add this to the PR ?

**Comment by berndhahnebach** [comment]
Do you really have a standard for welded profiles? In european these are not in a standard as one could weld whatever he would like to weld together if it fits in the welding standard. Usually you only weld a profile if you need a non standard geometry because welded profiles are much more expensive the hot rolled ones.

**Comment by berndhahnebach** [comment]
your changes are all about data. would you start the commit messages with "data"

**Comment by Andrej730** [comment]
Fixed commit names. Not sure about about welded profiles, summon @Moult 🙂

**Comment by Moult** [comment]
Welded profile names are correct, they follow the same convention as UB/UC.

**Comment by Moult** [comment]
From Australian engineer: "Yes, welded profiles come in standardized sizes"

**Comment by Moult** [comment]
I did a quick spot check and indeed it seems to address all the concerns from the FreeCAD forums.

File: data/profile_hollow.blt [hunk 1]
```diff
@@ -164,25 +168,29 @@ classes:
   parameters:
     defaults:
       l: 1000
-      type: 38x25RHS20
+      type: 75x25x2.5 RHS
     description:
+      ifr: inner fillet radius
+      ofr: outer fillet radius
```
Comment (berndhahnebach) [inline data/profile_hollow.blt]: the filletradius have to have the same short cut to work in geometric creation methods of the backends. I used ri and ro but I am not native englisch speaker. https://github.com/boltsparts/boltsparts/commit/cf748acf77ea969f612bcd0699ee84fde2c5f08a#diff-8c38c4eb402a7fd1ea39747f19c5bcff436645931fd45bd0671391b06a8d8f7bR246-R247 
I  do not mind to change it but we should make sure to use something handy. Best from a standard. How does ASIC call it?

**Comment by berndhahnebach** [comment]
> From Australian engineer: "Yes, welded profiles come in standardized sizes"

OK than we will add them as well.  Thanks very much for the explicit reinsurance in this regard.

Still it sounds strange to me from European workflows ins structural industry. ;-)

File: data/profile_hollow.blt (hunk 1 above)
Comment (Andrej730) [reply data/profile_hollow.blt]: In the specs I was working with it was actually called `ro` and `ri` too. 
`ifr` and `ofr` were just made up names and I've fixed that in https://github.com/boltsparts/boltsparts/pull/4/commits/81f6e812896acf1ea9c060f88ee6e43e563324a2

Comment (Moult) [reply data/profile_hollow.blt]: I checked in the Australian docs and couldn't find any special word for it.

**Comment by berndhahnebach** [comment]
I had to fix classids ... https://github.com/boltsparts/boltsparts/commit/e09893ce126c1dfffbf1f12ab0a75df4563f03c8 and added method to freecad backend to create ibeams without fillets. https://github.com/boltsparts/boltsparts/commit/dcadedc5ea43d17dc7e1979d19c0b560ac303c78

**Pull request merged**